spec/support/config/with_public_file_server.yml

```diff
@@ -0,0 +1,17 @@
+port: 3000
+port_reuse: true
+process_count: 1
+redis_url: "redis://localhost:6379"
+database_url: "mysql://root@localhost:3306/development_settings_test"
+
+session:
+  key: "myapp.session"
+  store: "encrypted_cookie"
+  expires: 10
+
+secrets:
+  description: "Store your test secrets credentials and settings here."
+
+public_file_server:
```

Comment:
This name `public_file_server` doesn't relate to the `static` pipe which is confusing to me.  Also, It may not be publicly facing.  Maybe rename it to `static_file_server`
...or...
What if we create a section for `pipes` settings?
```
pipes:
  static:
    headers:
      "Cache-Control": "private, max-age=7200"
```
We can move other pipe specific settings here.



Also move this section above `secrets` since it should be last.

Reply:
@drujensen Makes sense. I like the idea of per-pipe configuration. Let me update this.